### PR TITLE
[zh] Add faas/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
- Adds `content/zh/docs/faas/_index.md`
- **Preview**: https://deploy-preview-4617--opentelemetry.netlify.app/zh/docs/faas/

### CODE REVIEW

File: content/zh/docs/faas/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: 功能即服务（Faas）
+linkTitle: FaaS
+description: >-
+  OpenTelemetry 支持不同云供应商提供的各种功能即服务（Faas）的监控方法
```
Comment (windsonsea) [inline content/zh/docs/faas/_index.md:5]:
```suggestion
  OpenTelemetry 支持使用各种方法来监控不同云供应商提供的功能即服务（Faas）
```

File: content/zh/docs/faas/_index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+weight: 360
+default_lang_commit: 64fbfee6
+---
+
+功能即服务（FaaS）是云原生应用程序的重要无服务（Serverless）计算平台。
+但是，平台怪癖通常意味着这些应用程序与在 Kubernetes 或虚拟机上运行的应用程序具有略微不同的监控指导和要求。
```
Comment (windsonsea) [inline content/zh/docs/faas/_index.md:11]:
```suggestion
功能即服务（FaaS）是云原生应用的重要无服务（Serverless）计算平台。
但不同的平台有自己的风格特点，这通常意味着这些云原生应用与在 Kubernetes 或虚拟机上运行的应用相比，其监控指导和要求会略微不同。
```
how about to use `应用` for `application`, same below.

File: .cspell.yml [hunk 3]
```diff
@@ -59,6 +59,7 @@ words:
   - EMEA
   - erlang
   - errorf
+  - Faas
```
Comment (chalin) [inline .cspell.yml:62]: This addition to the dictionary shouldn't be necessary.

```suggestion
```

File: content/zh/docs/faas/_index.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: 功能即服务（Faas）
```
Comment (chalin) [inline content/zh/docs/faas/_index.md:2]:
```suggestion
title: 功能即服务（FaaS）
```

File: content/zh/docs/faas/_index.md (hunk 1 above)
Comment (my-git9) [reply content/zh/docs/faas/_index.md]: I feel that this suggestion deviates from the original intention.

Comment (my-git9) [reply content/zh/docs/faas/_index.md]: @reyang WDYT?

Comment (reyang) [reply content/zh/docs/faas/_index.md]: > I feel that this suggestion deviates from the original intention.

+1

File: content/zh/docs/faas/_index.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+title: 功能即服务（FaaS）
+linkTitle: FaaS
+description: >-
+  OpenTelemetry 支持不同云供应商提供的各种功能即服务（FaaS）的监控方法
+weight: 360
+default_lang_commit: 64fbfee6
+---
+
+功能即服务（FaaS）是云原生应用的重要无服务（Serverless）计算平台。
+但是，平台怪癖通常意味着这些应用与在 Kubernetes 或虚拟机上运行的应用具有略微不同的监控指导和要求。
```
Comment (reyang) [inline content/zh/docs/faas/_index.md:11]: “platform quirks” 直译好像会比较奇怪，试试 “平台的特异性”？

**Review by reyang — APPROVED** [review]

File: content/zh/docs/faas/_index.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/docs/faas/_index.md]: OK, let's keep original and mark it resolved

File: content/zh/docs/faas/_index.md (hunk 5 above)
Comment (windsonsea) [reply content/zh/docs/faas/_index.md:11]: 额，这个没改过来吗？

**Review by windsonsea — APPROVED** [review]
thanks

**Review by svrnm — APPROVED** [review]

**Pull request merged**